### DISCUSSION TITLE
Top/bottom padding

### DISCUSSION THREAD

**Discussion by dsparkplug** [ajaxorg/ace#5460] [Q&A] [answered]
For me, the first line of text seems really close to the top border. Is there a way to add padding to the top and/or bottom of the editor? I tried adding a top or margin-top value to the .ace_scroller class via css, but then the scroller misses the last line.

**Comment by nightwing** [comment, marked as answer]
You can use [editor.renderer.setScrollMargin](https://github.com/ajaxorg/ace/blob/55ca91f7cdb59abd6c4b5cdcab80c4283b0342d4/ace.d.ts#L724)

**Reply by dsparkplug** [reply]
Excellent, thanks. I couldn't find that in the wiki.
